test/dlc_tests/sanity/test_utility_installation.py

```diff
@@ -99,7 +99,7 @@ def test_utility_packages_using_import(training):
             ctx,
             f"import {package}; print({package}.__version__)",
             executable="python",
-        ).stdout.strip()
+        ).stdout.strip().strip().split()[-1]
```

Comment:
Do we need two `.strip()`'s here? Would it be possible/safer to use a regular expression like `r"\d+(\.\d+)+"` to find the version substring from the output?

Reply:
Replaced with regex